src/server/worker.cc

```diff
@@ -252,7 +252,8 @@ Status Worker::listenTCP(const std::string &host, uint32_t port, int backlog) {
     }
 
     evutil_make_socket_nonblocking(fd);
-    auto lev = NewEvconnlistener<&Worker::newTCPConnection>(base_, LEV_OPT_CLOSE_ON_FREE, backlog, fd);
+    auto lev =
+        NewEvconnlistener<&Worker::newTCPConnection>(base_, LEV_OPT_THREADSAFE | LEV_OPT_CLOSE_ON_FREE, backlog, fd);
```

Comment:
I wonder if `THREADSAFE` flag will affect performance in a explicit way?

Reply:
Yes, I guess it will affect the performance since it needs an extra lock and unlock operation. But it should be fine since the bottlenecks of accept connections won't be here. Let's me have a performance compare later.

Reply:
I tested on Ubuntu 22.04, it's no different. The performance of connecting is around 1.4W QPS.